ipatests/prci_definitions/nightly_latest.yaml

```diff
@@ -1685,7 +1685,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_ipa_cert_fix.py
+        test_suite: test_integration/test_ipa_cert_fix.py::TestIpaCertFix test_integration/test_ipa_cert_fix.py::TestCertFixKRA
```

Comment:
No need to specify each Test class, the file name is enough to execute all the test classes inside `test_ipa_cert_fix.py`. Same comment applies to all the yaml files.

Reply:
Done